File: python/enolib/elements/missing/missing_fieldset_entry.py
from .missing_value_element_base import MissingValueElementBase

class MissingFieldsetEntry(MissingValueElementBase):
    def __repr__(self):
        if self._key:
            return f"<class MissingFieldsetEntry key={self._key}>"

        return '<class MissingFieldsetEntry>'

File: ruby/lib/enolib/elements/missing/missing_fieldset_entry.rb
# frozen_string_literal: true

module Enolib
  class MissingFieldsetEntry < MissingValueElementBase
    def to_s
      if @key
        "#<Enolib::MissingFieldsetEntry key=#{@key}>"
      else
        '#<Enolib::MissingFieldsetEntry>'
      end
    end
  end
end
